src/main/java/com/graphaware/module/es/ElasticSearchWriter.java

```diff
@@ -40,10 +40,12 @@
  * A {@link ThirdPartyWriter} to Elasticsearch (https://www.elastic.co/).
  */
 public class ElasticSearchWriter extends BaseThirdPartyWriter {
+    public static final int MAXIMUM_CONSECUTIVE_ERRORS = 50;
```

Comment:
Make this much smaller please, like 5. And add a configuration option for increasing it.